targets/circuitplay-bluefruit.ld

```diff
@@ -0,0 +1,10 @@
+
+MEMORY
+{
+    FLASH_TEXT (rw) : ORIGIN = 0x00000000+0x26000, LENGTH = 796K /* SoftDevice S140. See https://learn.adafruit.com/introducing-the-adafruit-nrf52840-feather/hathach-memory-map */
```

Comment:
It may be clearer to describe the length with a formula too: `LENGTH = 1M - 0x26000`. That way it is immediately obvious where that length comes from.

Reply:
Done